mne/preprocessing/ica.py

```diff
@@ -987,6 +987,10 @@ def score_sources(self, inst, target=None, score_func='pearsonr',
                                  'number of time slices.')
             # auto target selection
             if isinstance(inst, BaseRaw):
+                # We need to pass inst, not self for defining sfreq, since the
+                # sfreq when fitting ica (--> self.info['sfreq']) can be
+                # different (usually lower) than the sfreq when scoring sources
+                # (e.g. using the original raw sfreq for this step):
```

Comment:
I would keep the comment short: something like `we pass inst, not self, because the sfreq of the data we apply ica to can be different`, all the other details are not necessary.

Reply:
Yes, that I have to learn, to KISS ;)